stories/Containers/SlidingOutPanels.stories.js

```diff
@@ -8,25 +8,25 @@ storiesOf('Sliding Out Panel', module).add('with default', () => (
 
 const images = [
     {
-        key: 1,
+        id: '1',
         alt: 'Display Image',
         imageLink:
             'https://stripe.com/img/docs/billing/hosted-invoice-page.png',
```

Comment:
Try to use different images to help make other people who view this component more clear on teh functionality

Reply:
change implemented, using different images as suggested

Reply:
image updated with custom images